modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements_controller.rb

```diff
@@ -83,20 +83,11 @@ def new_notice_of_disagreement
       auth_headers: request_headers,
       form_data: @json_body,
       source: request_headers['X-Consumer-Username'],
-      board_review_option: @json_body['data']['attributes']['boardReviewOption'],
-      api_version: api_version
```

Comment:
We definitely want to keep the api version saved on the model, whether it's v1 or v2.

Reply:
Ok. But, we were only doing this for NOD because it checked if it was 'v1' or 'v2'. We don't currently have a pattern for holding onto the API version. We just pass it in, or it defaults to 'v1'.

Reply:
I believe we have an example pattern with HLRs. Since we're adding the v2 controller, we should update both it and this v1 controller to store the correct api version.